src/realm/version_id.hpp

```diff
@@ -19,6 +19,10 @@
 #ifndef REALM_VERSION_ID_HPP
 #define REALM_VERSION_ID_HPP
 
+#ifdef _WIN32
+#define __STDC_LIMIT_MACROS
```

Comment:
Visual Studio gave warnings in Visual Studio without this define. It defines some C99 constants that aren't available by default in C++11. I didn't want to fix all this in this PR